src/Microsoft.Health.Fhir.Shared.Api.UnitTests/Features/Resources/Bundle/BundleHandlerTests.cs

```diff
@@ -286,7 +286,7 @@ public async Task GivenABundle_WhenProcessed_CertainResponseHeadersArePropagated
         }
 
         [Fact]
-        public async Task GivenABundle_WhenOneRequestProducesA429_SubsequentRequestAreSkipped()
+        public async Task GivenABundle_WhenOneRequestProducesA429_429IsRetriedThenSubsequentRequestAreSkipped()
```

Comment:
Is it worth adding a test that has a retry succeed to show the happy retry path?